frameworks/Java/javalin/javalin.dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gradle:4.7.0-jdk8
+USER root
+WORKDIR /javalin
+COPY build.gradle build.gradle
+COPY ./ ./
```

Comment:
In general, please avoid copying the entire working directory (`COPY ./ ./`) in favor of copying specific files or directories, as you do with `build.gradle` and `src`.  Do this for TFB at least, but I recommend it when using Docker in any context.

Advantages include:
* The dockerfile is more clear about what it requires to build correctly.
* You use Docker's build cache more effectively.
* You're less likely to accidentally include files in the Docker image that you didn't mean to include.

There may be other dockerfiles in this project that do `COPY ./ ./`, but I would like to fix those eventually and want to avoid adding more.